specification/app/data-plane/Microsoft.App.DynamicSessions/preview/2024-10-02-preview/examples/SessionResourceFiles_ListFilesMetadata.json

```diff
@@ -0,0 +1,23 @@
+{
+  "title": "SessionResourceFiles_ListFilesMetadata",
+  "operationId": "SessionResourceFiles_ListFilesMetadata",
+  "parameters": {
+    "api-version": "2024-10-02-preview",
+    "sessionId": "testidentidier"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "value": [
+          {
+            "name": "testfile",
```

Comment:
can you please add directory also in the list?

Reply:
Added Path param to the examples